x-pack/plugins/ml/common/types/inference.ts

```diff
@@ -79,3 +107,15 @@ export interface ModelPipelines {
 export interface InferenceConfigResponse {
   trained_model_configs: ModelConfigResponse[];
 }
+
+export function isClassificationTotalFeatureImportance(
+  summary: ClassificationTotalFeatureImportance | RegressionTotalFeatureImportance
+): summary is ClassificationTotalFeatureImportance {
+  return (summary as ClassificationTotalFeatureImportance).classes !== undefined;
+}
+
+export function isRegressionTotalFeatureImportance(
+  summary: ClassificationTotalFeatureImportance | RegressionTotalFeatureImportance
+): summary is RegressionTotalFeatureImportance {
+  return (summary as RegressionTotalFeatureImportance).importance !== undefined;
+}
```

Comment:
We also have `feature_importance.ts` in the same directory, can we move all additions to this file to `feature_importances.ts`?

Reply:
Updated here https://github.com/elastic/kibana/pull/78238/commits/e18bc414d34f7ca2611b2fa0cbcf0a2403d980a4